packages/shared/src/models-vis.ts

```diff
@@ -30,6 +30,9 @@ export enum ScaleType {
   Gamma = 'gamma',
 }
 
+export type ColorScaleType = Exclude<ScaleType, 'gamma'>;
+export type AxisScaleType = Exclude<ColorScaleType, 'sqrt'>;
```

Comment:
```suggestion
export type AxisScaleType = Exclude<ScaleType, 'sqrt' | 'gamma'>;
```

Might be clearer to remove the interdependency of both types